### DISCUSSION TITLE
关于使用点击input框，键盘弹起但光标消失的问题

### DISCUSSION THREAD

**Discussion by taixiaojingyan** [NervJS/taro#13576] [Q & A] [answered]
在项目里一个页面上有多个input,两个input框相互切换，触发了手机键盘弹起，但是光标消失了，需要再次点击才会在出现光标；安卓会出现问题，ios不会，taro版本是3.5.6，但是同样另一个3.5.6的taro项目里 多个input就没有这种问题；不知道大家有没有遇到过？

**Comment by taixiaojingyan** [comment, marked as answer]
目前了解到（Input  type="number"的时候就不会有这种问题，而且在tabBar内也没有问题，只有在单独的页面的时候才会有这种问题）

**Reply by taixiaojingyan** [reply]
解决了 解决了

**Reply by a993897020** [reply]
怎么解决的呀

**Reply by taixiaojingyan** [reply]
https://juejin.cn/post/7220244579672571964   解决方法写在了这里

**Reply by a993897020** [reply]
你这种情况我试过了，还是会一直失焦聚焦，我想到另一种方案，就是用View在input的层级之上（z-index），这样点击事件就作用于View身上，通过点击View给input设置焦点，从而解决这个问题

**Reply by taixiaojingyan** [reply]
一会儿我试试，我的项目用我上面的那种方法可行的，问题解决了

**Reply by taixiaojingyan** [reply]
刚试了一下  对我的项目不太友好，我封装的form表单 不同类型的话 往里加蒙层的操作有些繁琐了
